### PR TITLE
Configure learning rate schedule with sticker-train options

### PR DESCRIPTION
**Pull request by danieldk**
This moves the learning rate settings out of the configuration file.

This is also the last larger planned change before branching release-0.3.

### CODE REVIEW

File: sticker-utils/src/bin/sticker-train.rs [hunk 1]
```diff
@@ -5,29 +5,80 @@ use std::io::BufReader;
...
+
 impl TrainApp {
     fn new() -> Self {
         let matches = sticker_app("sticker-train")
+            .arg(
+                Arg::with_name(INITIAL_LR)
+                    .long("lr")
+                    .value_name("LR")
+                    .help("Initial learning rate")
+                    .default_value("0.01"),
```
Comment (sebpuetz) [inline sticker-utils/src/bin/sticker-train.rs:59]: Iirc, 0.01 is a fairly high value, are you using that number in some models?

**Review by sebpuetz — APPROVED** [review]

File: sticker-utils/src/bin/sticker-train.rs (hunk 1 above)
Comment (danieldk) [reply sticker-utils/src/bin/sticker-train.rs:59]: It's my default for RNN models.

Comment (sebpuetz) [reply sticker-utils/src/bin/sticker-train.rs:59]: I see, I was probably thinking about the clause models. I think i used 0.005 or 0.001 on those.

Comment (danieldk) [reply sticker-utils/src/bin/sticker-train.rs:59]: The convolution models typically even diverge with 0.01, but I guess RNNs are currently the best default option. It's definitely something we should add to the documentation.

**Pull request merged**